packages/storybook/src/stories/ino-popover/ino-popover.stories.ts

```diff
@@ -34,84 +83,50 @@ export const Playground = (args) => html`
-  hideOnBlur: false
-};
-Playground.argTypes = {
-  placement: {
-    control: {
-      type: 'select',
-    },
-    options: ['top', 'right', 'bottom', 'left'],
-  },
-  trigger: {
-    control: {
-      type: 'select',
-    },
-    options: [
-      'mouseenter',
-      'focus',
-      'click',
-      'mouseenter focus',
-      'mouseenter click',
-      'focus click',
-      'mouseenter focus click',
-    ],
-  },
-  colorScheme: {
-    control: {
-      type: 'select',
-    },
-    options: ['primary', 'secondary', 'light', 'transparent'],
-  },
-};
+  
+`);
 
-export const Placement = () => html`
-  <ino-popover placement="left" for="popover-positions-target">
-    <div class="popover-content">This is a simple popover on the left.</div>
-  </ino-popover>
-  <ino-popover placement="right" for="popover-positions-target">
-    <div class="popover-content">This is a simple popover on the right.</div>
-  </ino-popover>
-  <ino-popover placement="top" for="popover-positions-target">
-    <div class="popover-content">This is a simple popover on the top.</div>
-  </ino-popover>
-  <ino-popover placement="bottom" for="popover-positions-target">
-    <div class="popover-content">This is a simple popover on the bottom.</div>
-  </ino-popover>
-  <ino-button class="placement-button" id="popover-positions-target"
-    >Popover</ino-button
-  >
-`;
+export const Playground = template.generatePlaygroundStory();
+export const Placement = template.generateStoryForProp('placement', 'right', {
+  id: 'popover-target2',
+  for: 'popover-target2',
+});
+export const AttachToBody = template.generateStoryForProp('attachToBody', true, {
+  id: 'popover-target3',
+  for: 'popover-target3',
+});
+export const HideOnBlur = template.generateStoryForProp('hideOnBlur', true, {
+  id: 'popover-target4',
+  for: 'popover-target4',
+});
+export const HideOnEsc = template.generateStoryForProp('hideOnEsc', true, {
+  id: 'popover-target5',
+  for: 'popover-target5',
+});
+export const Distance = template.generateStoryForProp('distance', 30, {
+  id: 'popover-target6',
+  for: 'popover-target6',
+});
+export const ColorScheme = template.generateStoryForProp('colorScheme', 'secondary', {
+  id: 'popover-target7',
+  for: 'popover-target7',
+});
+export const Trigger = template.generateStoryForProp('trigger', 'click', {
+  id: 'popover-target9',
+  for: 'popover-target9',
+});
 
-export const Interactions = () => html`
-  <ino-button id="popover-non-interactive">Non-Interactive content</ino-button>
-  <ino-popover
-    for="popover-non-interactive"
-    trigger="click"
-    color-scheme="transparent"
-  >
-    <div class="popover-content">
-      I'm not interactive. I will close on click. I should only be used for
-      non-interactive content like this text.
-    </div>
-  </ino-popover>
+
+
+const templateInteractive = new TemplateGenerator<Components.InoPopover>(
+  'ino-popover',
+  args => html`
   <ino-popover
     trigger="click"
     placement="bottom"
```

Comment:
Can you change the placement to left or right? On `placement="bottom"` the popover is turncated.

![image](https://user-images.githubusercontent.com/23154336/173511290-b65eccf3-891d-421f-9395-4b5f816e2ce9.png)

Reply:
Why I resolved this^^